### ISSUE TITLE
how to calculate the preprocessing time

### ISSUE THREAD

**Issue by yeeinn** [drprojects/superpoint_transformer#151]
How to calculate the time spent on preprocessing. it is 12.4min of s3dis in your paper .

**Comment by drprojects** [comment]
The preprocessing times for the train, val, and test set all appear in the logs (see progress bars) when instantiating a dataset for the first time and preprocessing is triggered.

Otherwise, you can time whatever piece of code you are interested in with:

```python
import torch
from time import time

torch.cuda.synchronize()
start = time()

# stuff happens...

torch.cuda.synchronize()
end = time()

print(end - start)
```
